src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -6087,6 +6087,15 @@ components:
           $ref: '#/components/schemas/ConsentCode'
         duosId:
           $ref: '#/components/schemas/DuosId'
+        dataAccessControlGroups:
```

Comment:
is this the best name for this new field?

Reply:
I like this field name!  It matches the language we've settled on in the TDR UI: https://github.com/DataBiosphere/jade-data-repo-ui/pull/1573

The existing endpoint to add DACs to an existing snapshot was pigeonholed into language that focuses on auth domains, so it's nice to be able to move away from that: https://data.terra.bio/swagger-ui.html#/snapshots/addSnapshotAuthDomain

Reply:
@okotsopoulos Do you think the name dataAccessControlGroups would be less confusing to users because they will be used to using the TDR UI or authDomains would be less confusing to users because they are used to using the API? Or either way is okay because the API endpoint's description uses the language "data access control groups"? Are we looking to rename this endpoint in the future?